clang/lib/Driver/Driver.cpp

```diff
@@ -4158,18 +4158,12 @@ class OffloadingActionBuilder final {
       // supported GPUs.  sm_20 code should work correctly, if
       // suboptimally, on all newer GPUs.
       if (GpuArchList.empty()) {
-<<<<<<< HEAD
-        if (ToolChains.front()->getTriple().isSPIROrSPIRV())
-          GpuArchList.push_back(CudaArch::Generic);
-        else
-=======
-        if (ToolChains.front()->getTriple().isSPIRV()) {
+        if (ToolChains.front()->getTriple().isSPIROrSPIRV()) {
           if (ToolChains.front()->getTriple().getVendor() == llvm::Triple::AMD)
             GpuArchList.push_back(CudaArch::AMDGCNSPIRV);
           else
             GpuArchList.push_back(CudaArch::Generic);
-        } else {
->>>>>>> 9acb533c38be833ec1d8daa06e127a9de8f0a5ef
+	} else {
```

Comment:
```suggestion
	    } else {
```
indentation fix please